.github/workflows/ci.yml

```diff
@@ -19,17 +19,113 @@ jobs:
         with:
           coverage: none
           php-version: '8.2'
-      - run: composer validate --strict --no-check-lock
+      - run: composer validate --strict --no-check-lock --ansi
 
   static_analysis:
-      name: Static analysis
-      runs-on: ubuntu-latest
-      steps:
-          - uses: actions/checkout@v4
-          - uses: shivammathur/setup-php@v2
-            with:
-                coverage: none
-                php-version: '8.2'
-          - name: Install dependencies
-            run: composer update --ansi --no-progress --prefer-dist --no-interaction
-          - run: vendor/bin/phpstan analyze
+    name: Static analysis
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: shivammathur/setup-php@v2
+        with:
+          coverage: none
+          php-version: '8.2'
+      - run: composer update --ansi --no-progress --prefer-dist --no-interaction
+      - run: vendor/bin/phpstan analyze --ansi --no-progress
+
+  Official:
+    name: "🛡️ ${{ matrix.name }}"
+    uses: "./.github/workflows/test.yml"
+    with:
+      name: ${{ matrix.name }}
+      driverRepoUrl: ${{ matrix.driverRepoUrl }}
+      php: ${{ matrix.php }}
+      setUpCmd: ${{ matrix.setUpCmd }}
+      testCmd: ${{ matrix.testCmd }}
+      tearDownCmd: ${{ matrix.tearDownCmd }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: "BrowserKit (http client)"
+            driverRepoUrl: "https://github.com/minkphp/MinkBrowserKitDriver.git"
+            php: "7.2"
+            # language=bash
+            testCmd: ./vendor/bin/phpunit --colors=always --testdox --configuration ./phpunit.http_client.xml
+
+          - name: "BrowserKit (http kernel)"
+            driverRepoUrl: "https://github.com/minkphp/MinkBrowserKitDriver.git"
+            php: "7.2"
+            # language=bash
+            testCmd: ./vendor/bin/phpunit --colors=always --testdox
+
+          - name: "Selenium2"
+            driverRepoUrl: "https://github.com/minkphp/MinkSelenium2Driver.git"
+            php: "7.2"
+            # language=bash
+            setUpCmd: |
+              export SELENIUM_IMAGE=selenium/standalone-firefox:2.53.1
+              docker compose up --wait --quiet-pull
+              curl --retry 5 --retry-all-errors --retry-delay 1 --max-time 10 --head -X GET http://localhost:4444/wd/hub/status
+            # language=bash
+            testCmd: |
+              export WEB_FIXTURES_BROWSER=firefox
+              export DRIVER_MACHINE_BASE_PATH=/fixtures/
+              export WEB_FIXTURES_HOST=http://host.docker.internal:8002
+              ./vendor/bin/phpunit --colors=always --testdox
+            # language=bash
+            tearDownCmd: |
+              docker compose logs --no-color --no-log-prefix --timestamps selenium > ./logs/selenium.docker.log
```

Comment:
Could you please update this part to reflect build script changes made in the https://github.com/minkphp/MinkSelenium2Driver/pull/406?

Reply:
I'm not sure what you mean, it's already using `docker compose` to extract logs (and the setUpCmd also uses it).

Reply:
Ah, I thought that you'd copy-pasted code fragments from the driver's build script. Apparently you didn't. Anyway, there is now a `docker-compose.yml` file in the driver's repository that you could use.

I wonder how you're able to correctly start Selenium's Docker image with only `docker compose up --wait --quiet-pull` command instead of `docker run --net host --name selenium --volume /dev/shm:/dev/shm --volume ./vendor/mink/driver-testsuite/web-fixtures:/fixtures --shm-size 2g selenium/standalone-firefox:${{ matrix.selenium_version }} &> ./logs/selenium.log &` and still make fixtures discoverable from the Selenium ?

Reply:
> I wonder how you're able to correctly start Selenium's Docker image with only...

All the parts you mentioned after `docker run` are in the `docker-compose.yml` file. So `docker compose up` is effectively doing the same thing as `docker run`.

Note that `docker-compose up` has been replaced with `docker compose up` (in case that's unclear).

Reply:
There were no `docker-compose.yml` file in the MinkSelenium2Driver repo until the https://github.com/minkphp/MinkSelenium2Driver/pull/406. I wasn't able to pinpoint the exact commit where you did this, but initially you've used a `docker run` command, but then (probably once you've noticed `docker-compose.yml` file in the MinkSelenium2Driver repo) you've replaced it with `docker compose` command. 

If I'm not mistaken, then no further actions are needed on your side based on my original question.